### PR TITLE
Hds 1955 - Automatically Closing Notification

### PR DESCRIPTION
**Pull request by arpargo**
## Description

Fixing an overlapping example notifications for automatically closing Notifications.

## Related Issue

[HDS-1955](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1955)

Closes #

## Motivation and Context

On smaller screensizes, notifications would stack on top of eachother and especially on mobile examples, you'd never see the other notification as they both open and close at the same time.

## How Has This Been Tested?
Locally

[HDS-1955]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-1955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hds-core",
-  "version": "3.0.0-alpha.0",
```
Comment (mrTuomoK) [inline packages/core/package.json:3]: Aren't these version changes here a bit weird since they are already in `development`. Did you rebase?

Comment (mikkojamG) [reply packages/core/package.json:3]: Seeing extra commit https://github.com/City-of-Helsinki/helsinki-design-system/pull/1139/commits/2377a5a2d537c36457469039f6459d11c284800f that should be dropped

**Review by mikkojamG — APPROVED** [review]

File: packages/core/package.json (hunk 1 above)
Comment (arpargo) [reply packages/core/package.json]: Rebased to most current development and that fixed the issue of extra commit appearing on this PR.

**Review by harriplappalainen — APPROVED** [review]
Solid fix to the problem 🏅

**Pull request merged**